js/leo.js

```diff
@@ -18,6 +18,12 @@ var topicsForTypeClient;
 
 var select;
 
+var intervalFlag = false;
+var initROSinterval;
+var last_selection;
```

Comment:
Don't mix up different naming cases for variables
```suggestion
var lastSelection;
```

Reply:
Done for all variables.